hack/apply_folder.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+function error_exit() {
+    msg=${1}
+    >&2 echo "[Error] ${msg}"
+    exit 1
+}
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+BASE_DIR=$(realpath "${THIS_DIR}/..")
+
+echo
+echo ">> Running apply_folder hack script"
+echo "   --------------------------------"
+echo
+if [ -n "${OPENSHIFT_CI:-}" ]; then
+    source "${THIS_DIR}/setup_prow_env.sh"
+fi
+
+# Setup defaults
+TERRAFORM_D_DIR=${TERRAFORM_D_DIR:-"${HOME}"}
+WORK_DIR=${WORK_DIR:-"${BASE_DIR}/playground"}
+
+echo "[INFO] Will apply folder."
+echo "[INFO] - WORK_DIR: ${WORK_DIR}"
+echo "[INFO] - TERRAFORM_D_DIR: ${TERRAFORM_D_DIR}"
+echo "[INFO] - THIS_DIR: ${THIS_DIR}"
+echo "[INFO] - BASE_DIR: ${BASE_DIR}"
+
+if [[ ! -d ${WORK_DIR} ]]; then
+    error_exit "can't load WORK_DIR"
+fi
+
+set -x
```

Comment:
for consistency with other declerations, here and elsewhere:
```suggestion
set -o xtrace
```